pylint/reporters/text.py

```diff
@@ -193,17 +190,21 @@ class ColorizedTextReporter(TextReporter):
     """Simple TextReporter that colorizes text output"""
 
     name = "colorized"
-    COLOR_MAPPING = {
-        "I": ("green", None),
-        "C": (None, "bold"),
-        "R": ("magenta", "bold, italic"),
-        "W": ("magenta", None),
-        "E": ("red", "bold"),
-        "F": ("red", "bold, underline"),
-        "S": ("yellow", "inverse"),  # S stands for module Separator
+    COLOR_MAPPING: ColorMappingDict = {
+        "I": MessageStyle("green", ()),
```

Comment:
It might make sense to add `()` as default for `MessageStyle.style`. Only downside, you'll need to import `NamedTuple` from `typing_extensions` instead. Support for default values was only added in `3.6.1`.
https://docs.python.org/3/library/typing.html#typing.NamedTuple

Reply:
I don't think `NamedTuple` is supported by `typing_extensions`. See their [README](https://github.com/python/typing/blob/master/typing_extensions/README.rst)

Reply:
😕 Didn't know that